### DISCUSSION TITLE
Add only keep latest caching option to save disk space

### DISCUSSION THREAD

**Discussion by panghaoyuan** [vercel/turborepo#1723] [Ideas]
Currently computation caching is cumulative. It will affects the upload and download speed of the cache in the CI environment like gitlab and will also significantly increase the execution time of the CI.

Is it possible to add one option to only keep latest computation cache?

**Comment by anthonyshew** [comment]
A TTL should be implemented server-side in accordance with web standards. This isn't something that the `turbo` CLI can dictate since flags/options are mutable as a client.
